openhtf/util/validators.py

```diff
@@ -75,9 +76,30 @@ def create_validator(name, *args, **kwargs):
 
 _identity = lambda x: x
 
+
+class ValidatorBase(object):
```

Comment:
This implies that validators must be classes, but they just need to be callables. We use classes so there are attributes, but a validator like this is also fine: `lambda value: value == 1`